src/it/it-abstract-versions-report-001/verify.groovy

```diff
@@ -0,0 +1,15 @@
+dependencyUpdatesReport = new File( basedir, "target/site/dependency-updates-report.html" )
+
+assert dependencyUpdatesReport.exists(  )
+
+// some basic (=not comprehensive) checks on the contents of the dependency report
+assert dependencyUpdatesReport.text =~ /\b1\.1\.0-2\b/  // current version of the dependency
+assert dependencyUpdatesReport.text =~ /\b3\.0\b/       // latest major available version
+
+pluginUpdatesReport = new File( basedir, "target/site/plugin-updates-report.html" )
+
+assert pluginUpdatesReport.exists(  )
+
+// some basic (=not comprehensive) checks on the contents of the plugin report
+assert pluginUpdatesReport.text =~ /\b2\.5\.3\b/        // current version of the plugin
+assert pluginUpdatesReport.text =~ /\b3\.0\.0-M6\b/     // latest major available version
```

Comment:
I would change this assertions to versions of dumy plugin.

Used `3.0.0-M6` is the latest version of m-release-p - when new version will be released - what will be ?

Reply:
Sorry, my bad. Corrected.